src/vivarium/framework/artifact/artifact.py

```diff
@@ -214,15 +214,15 @@ def __str__(self):
         return out
 
 
-def _to_tree(keys: List[str]) -> Dict[str, Dict[str, List[str]]]:
-    out = defaultdict(lambda: defaultdict(list))
+def _to_tree(keys: list[str]) -> dict[str, dict[str, list[str]]]:
+    out: defaultdict[str, dict[str, list[str]]] = defaultdict(lambda: defaultdict(list))
```

Comment:
This was weird to me that I needed to type hint the args to a dict that can take any structure.

Reply:
Yeah, my brain hurts right now and it's hard for me to figure out why that might be (or even if what you've done is correct)